centos/8/Dockerfile

```diff
@@ -6,14 +6,9 @@ RUN yum -y install wget yum-utils
 
 # Enable extra repositories
 RUN yum -y install epel-release
-# added PowerTools as suggested at:
-#   https://fedoraproject.org/wiki/EPEL
-RUN yum config-manager --set-enabled PowerTools
```

Comment:
EPEL packages may depend on powertools, it is declared on the given web page:

> on CentOS 8 it is recommended to also enable the powertools repository since EPEL packages may depend on packages from it:
> ```
> # dnf config-manager --set-enabled powertools
> ```

We should just write it [lowercased](https://www.reddit.com/r/CentOS/comments/jd7x3d/how_to_enable_powertools_in_centos_stream/).

Reply:
Sure, that's better way, I failed to find such solution by myself, thanks.